internal/gossip/gossip_redis.go

```diff
@@ -54,7 +53,10 @@ func (g *GossipRedis) Start() error {
 						select {
 						case ch <- msg.data:
 						default:
-							fmt.Println("dropped message", msg.key, string(msg.data))
+							g.Logger.Debug().WithFields(map[string]interface{}{
```

Comment:
I feel these should be error logs.  Dropping a message here would be detrimental to Refinery's operations.

Reply:
Because of cleanup, it will add latency but you won't lose data. I do agree that debug is not right -- but what do you think of Warn instead of Error?